### PR TITLE
Upgrade Universal Viewer (should fix #93)

### PR DESCRIPTION
**Pull request by geekscruff**
This PR should fix issues with Universal Viewer. It ports the last Hyrax code for Universal Viewer (slated for release in hyrax 3.x), removing pul_uv_rails which is no longer maintained and upgrading universal viewer to v3 deployed via yarn.

There are changes to the Dockerfile for hyrax to install yarn.

I also noticed some inconsistency in the fits version, so I changed that and removed a redundant environment for fits in docker-compose.

Once deployed to the demo/staging site, this should be tested with IE and Edge.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -105,6 +105,9 @@ services:
   web:
     <<: *app
     command: bash -c "/bin/docker-entrypoint.sh"
+    environment:
+      - VIRTUAL_HOST=nims.docker
```
Comment (Steven-Eardley) [inline docker-compose.yml]: These could be inside the .env file (the .env.template file)

File: .env.template [hunk 2]
```diff
@@ -47,7 +48,7 @@ GA_CLIENT_EMAIL=
 
 # Hyrax application setting
 DERIVATIVES_PATH=/shared/derivatives/
-FITS_PATH=/fits/fits-1.0.5/fits.sh
+FITS_PATH=/fits/fits-1.3.0/fits.sh
```
Comment (Steven-Eardley) [inline .env.template]: shall we remove the version from the fits path, so we can install any version we like without this faff?

File: hyrax/config/initializers/hyrax.rb [hunk 3]
```diff
@@ -94,7 +94,7 @@
   # config.redis_namespace = "hyrax"
 
   # Path to the file characterization tool
-  config.fits_path = ENV['FITS_PATH'] || "/fits/fits-1.0.5/fits.sh"
+  config.fits_path = ENV['FITS_PATH'] || "/fits/fits-1.3.0/fits.sh"
```
Comment (Steven-Eardley) [inline hyrax/config/initializers/hyrax.rb]: versioned fits path, as above

File: .env.template (hunk 2 above)
Comment (paulwalk) [reply .env.template]: I think it makes sense to remove the versioning from the path - this also causes problems with the separate data import script which invokes fits. Perhaps a symlink approach?

File: docker-compose.yml (hunk 1 above)
Comment (geekscruff) [reply docker-compose.yml]: yes, those are only because I use dory (which is ace btw) so I will move them

File: .env.template (hunk 2 above)
Comment (geekscruff) [reply .env.template]: agreed, I'll have a look at that

**Pull request merged**